### ISSUE TITLE
Support ImageBrownDistortionModel in ogre2

### ISSUE THREAD

**Issue by mahiuchun** [gazebosim/gz-rendering#697]
## Desired behavior
ImageBrownDistortionModel is supported in "ogre2".

## Alternatives considered
N/A

## Implementation suggestion
I'm not very familiar.

## Additional context
It appears that whenever there is `<distortion>` in a camera sensor, gz-sensors would attempt to create an ImageBrownDistortionModel from the rendering engine. "ogre2" does not support ImageBrownDistortionModel right now while "ogre" supports it.

**Comment by TanguyTheOtter** [comment]
I would also be interested by this feature, could one of the gazebo devs provide some tips on how this could work? I'd be happy to try to implement and test this.

**Comment by scpeters** [comment]
as we have already dropped support for ogre1.9 on macOS in our development branches (see https://github.com/osrf/homebrew-simulation/pull/3394) and are discussing whether to do so for Ubuntu as well, I think this issue has heightened importance as image distortion is not currently supported on ogre 2.3

**Comment by iche033** [comment]
yeah it'd be good to get this implemented. We'll need some help from the community to tackle this. The notes in https://github.com/gazebosim/gz-rendering/issues/697#issuecomment-1678030282 could probably already be a good starting point to feed to AI for initial code generation. The [render_pass](https://github.com/gazebosim/gz-rendering/blob/15a0455307f002e9a5e1415198ebbd7ab5731c1b/test/integration/render_pass.cc#L370) integration test can then be enabled for `ogre2` for verifying results.
